src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -47,33 +48,20 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-
-        Query groupQuery = new Query(GROUP_KIND);
-        PreparedQuery preparedGroupQuery = datastore.prepare(groupQuery);
-
-        ImmutableList.Builder<ImmutableMap<String, String>> groupsListBuilder = new ImmutableList.Builder<>();
-        for (Entity group : preparedGroupQuery.asIterable()) {
-            if (defaultGroup == null) {
```

Comment:
you're not changing defaultGroup anymore so it'll always be null, and doPost seems to use it. not sure if that would be an issue